Rubberduck.Parsing/Inspections/Resources/InspectionsUI.resx

```diff
@@ -766,4 +766,16 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="CodeInspectionSeverity_Warning" xml:space="preserve">
     <value>Warning</value>
   </data>
+  <data name="EmptyElseBlockInspectionResultFormat" xml:space="preserve">
+    <value>Else block has no executable statements</value>
+  </data>
+  <data name="EmptyElseBlockInspectionMeta" xml:space="preserve">
+    <value>An Else block should have executable statements</value>
```

Comment:
The meta string is where we get to explain the rationale behind an inspection. Could be expanded a bit (with punctuation!).

> An empty else block serves no purpose and can be confusing for the maintainer, which is left wondering whether that execution path was intended to do something at one point. Clear code only contains executable execution paths.

Something like that.